books/RayTracingTheRestOfYourLife.html

```diff
@@ -736,10 +739,10 @@
 <div class='together'>
 After spending enough time with PDFs you might start referring to a PDF as the probability that a
 variable $r$ is value $x$, _i.e._ $p(r=x)$. Don't do this. For a continuous function, The
-probability that a variable is a specific value is always zero. If you ask "given the PDF $p(r)$,
-what are the odds that $r=1$"? For all PDFs this value is always zero. A PDF can only tell you the
+probability that a variable is a specific value is always zero. A PDF can only tell you the
 probability that a variable will fall within an interval. If the interval you're checking against is
```

Comment:
Suggestion: “... probability that a variable will fall within a given interval of non-zero width.”